src/main/java/fr/esgi/al5/tayarim/services/FactureService.java

```diff
@@ -230,9 +230,17 @@ public FactureDto sendFacture(@NonNull Long id) {
     Proprietaire proprietaire = facture.getProprietaire();
 
     System.out.println("Facture envoyée par mail");
+
+    Storage storage = TayarimApplication.bucket.getStorage();
+
+    BlobInfo blobInfo = BlobInfo.newBuilder(TayarimApplication.bucket.getName(),
+        "Factures/facture_" + facture.getNumeroFacture() + ".pdf").build();
+
+    Blob blob = storage.get(blobInfo.getBlobId()).toBuilder().setContentType("application/pdf").build();
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 104).